nova_vm/src/ecmascript/abstract_operations/testing_and_comparison.rs

```diff
@@ -119,3 +152,128 @@ pub(crate) fn same_value_non_number<T: Copy + Into<Value>>(_agent: &mut Agent, x
     // 7. If x is y, return true; otherwise, return false.
     todo!()
 }
+
+/// [7.2.13 IsLessThan ( x, y, LeftFirst )](https://tc39.es/ecma262/#sec-islessthan)
+///
+/// The abstract operation IsLessThan takes arguments x (an ECMAScript language
+/// value), y (an ECMAScript language value), and LeftFirst (a Boolean) and
+/// returns either a normal completion containing either a Boolean or undefined,
+/// or a throw completion. It provides the semantics for the comparison x < y,
+/// returning true, false, or undefined (which indicates that at least one
+/// operand is NaN). The LeftFirst flag is used to control the order in which
+/// operations with potentially visible side-effects are performed upon x and y.
+/// It is necessary because ECMAScript specifies left to right evaluation of
+/// expressions. If LeftFirst is true, the x parameter corresponds to an
+/// expression that occurs to the left of the y parameter's corresponding
+/// expression. If LeftFirst is false, the reverse is the case and operations
+/// must be performed upon y before x. It performs the following steps when
+/// called:
+pub(crate) fn is_less_than<const LEFT_FIRST: bool>(
+    agent: &mut Agent,
+    x: impl Into<Value> + Copy,
+    y: impl Into<Value> + Copy,
+) -> JsResult<Value> {
```

Comment:
**nitpick:** I think it might be worthwhile to leave out the technical `undefined` property of this and return a `JsResult<Option<bool>>` instead. Converting to a `Value` from there is fairly trivial from there after all.